.github/workflows/ci.yaml

```diff
@@ -25,18 +25,17 @@ jobs:
   test:
     uses: bazel-contrib/.github/.github/workflows/bazel.yaml@646899086d7aaee8e532540480f3e91e00596234 # 11 Dec 2024
     with:
-      folders: '[".", "e2e/smoke", "e2e/repository-rule-deps", "e2e/system-interpreter", "examples/uv_pip_compile"]'
+      # NB: the root folder is tested with Aspect Workflows on BuildKite, see /.aspect/workflows/config.yaml
+      folders: '["e2e/smoke", "e2e/repository-rule-deps", "e2e/system-interpreter", "examples/uv_pip_compile"]'
```

Comment:
Do you want me to wire up any of the subworkspaces to run under workflows? I'm not sure we could do the bzlmod/workspace switch easily though, so it would only run one variant.

Reply:
possibly in a follow-up yes

Reply:
as discussed in meeting - let's not bother with this anytime soon. We got the value we want here already, and will always have some amount of GHA on this repo